content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -0,0 +1,480 @@
+
+<details>
+
+### How are experiments queued?
+
+Queued experiments are created similar to
+[Git stash](https://www.git-scm.com/docs/git-stash). The last experiment queued
+is found in `.git/refs/exps`, and earlier ones are in its [reflog].
+
+[reflog]:
+  https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefreflogareflog
+
+</details>
+
+```dvc
+$ dvc exp run --queue -S units=10
+Queued experiment '1cac8ca' for future execution.
+$ dvc exp run --queue -S units=64
+Queued experiment '23660bb' for future execution.
+$ dvc exp run --queue -S units=128
+Queued experiment '3591a5c' for future execution.
+$ dvc exp run --queue -S units=256
+Queued experiment '4109ead' for future execution.
+```
+
+> `dvc exp show` will mark queued experiments with an asterisk `*`.
+
+Each experiment is derived from the workspace at the time it's queued. If you
+make changes in the workspace afterwards, they won't be reflected in queued
+experiments (once run).
+
+<details>
+
+### How are queued experiments isolated? (Temporary directories)
+
+To guarantee that queued experiments derive from their original workspace, DVC
+creates a copy of it in `.dvc/tmp/exps/`, where the experiment will run. All
+these workspaces share the main project <abbr>cache</abbr>.
+
+If you want to isolate an experiments this way without queuing it, you can use
+the `--temp` option. This allows you to continue working while a long experiment
+runs.
+
+```dvc
+$ nohup dvc exp run --temp &
+[1] 30473
+nohup: ignoring input and appending output to 'nohup.out'
+```
+
+> The above example creates a `nohup.log` file in the original workspace with
+> the output of the DVC process.
+
+Note that Git-ignored files/dirs are explicitly excluded from queued/temp runs
+to avoid committing unwanted files into final ([persisted]) experiments.
+
+[persisted]: /doc/user-guide/experiment-management#persistent-experiments
+
+</details>
+
+Run them all one-by-one with the `--run-all` flag. The order of execution is
+independent of their creation order.
+
+```dvc
+$ dvc exp run --run-all
+```
+
+> ⚠️ Note that only tracked files and directories will be included in
+> `--queue/temp` experiments. To include untracked files, stage them with `git
+> add` first (before `dvc exp run`) and `git reset` them afterwards.
+
+To remove all experiments from the queue and start over, you can use
+`dvc exp remove --queue`. 📖 See [Cleaning-Up Experiments][clean-up] for
+details.
+
+[clean-up]: /doc/user-guide/experiment-management/cleaning-up-experiments
+
+### Running experiments in parallel
+
+DVC allows to run queued experiments in parallel by specifying a number of
+execution processes (`--jobs`):
+
+```dvc
+$ dvc exp run --run-all --jobs 4
+```
+
+> Note that since each experiment runs in an independent temporary directory,
+> common <abbr>stages</abbr> across experiment pipelines are always run
+> separately even if they share param values or other dependencies.
```

Comment:
Is this correct? I thought all temp dirs share the same cache. Cc @pmrowla thanks

Reply:
They are executed in their own workspaces, so stages are run separately.

They do share the original workspace's cache directory, so if run cache is available from a previous stage run, that cache entry would be reused to skip execution. But that's an implementation detail, and it's still true either way that the stage run is evaluated separately for each workspace.

Reply:
Hmm. It comes down to what we're trying to say with "run separately". I think it can be misunderstood as they always run (even if there's a matching run-cache entry) which is not correct it seems. That would only happen in some parallel runs right? I'll rephrase... ⏳ 

Idk if this is an implementation detail, the run-cache is a user-facing feature. We have it documented and mentioned in a few places (granted, not a lot but still).

Reply:
Updated in 7a8f155f.

Reply:
but I wonder whether we should just delete this whole note.